### DISCUSSION TITLE
sdk 32 to 33 doesn't change

### DISCUSSION THREAD

**Discussion by jeuxdaily** [airsdk/Adobe-Runtime-Support#633] [Q&A]
## Problem Description

Describe your problem in detail. Include the following information:

* Which AIR SDK did you use? Hint: always try the latest version from [Adobe Labs](http://labs.adobe.com/downloads/air.html).
* Describe the affected environment: Flash vs. AIR, Android vs. iOS, all devices vs. just specific models.
* Can you reproduce it yourself, or did you get reports from other users? If so, how many users are affected (percentage)?
* If applicable, link to the Starling/Feathers issues that brought you here.

Furthermore, **always** add a link to one or more analog [Adobe Tracker](https://tracker.adobe.com) issues. This repository is not a replacement for Adobe's bug tracker!

## Steps to Reproduce

Tell us how this issue can be reproduced. Seriously: if you want this issue to be fixed, this is the part you **really** need to get nailed.
Give us a minimal example:

* As little code as possible. Get to the root of the problem!
* As few dependencies as possible. Did the problem pop up with Feathers? Try to reproduce it with Starling only.
* Remove any ANEs (except if they are vital for the problem).
* Include all necessary files (e.g. textures).

## Known Workarounds

Just in case others run into this – do you happen to know of a workaround? Otherwise, just remove this section altogether.

**Comment by jeuxdaily** [comment]
hello

I have a problem when I compile to apk

in file app.xml this line

<application xmlns="http://ns.adobe.com/air/application/32.0">

doesn't change to 33

I have sdk 33

so I run two file

adt.bat

![adt_cp](https://user-images.githubusercontent.com/14032996/63125092-bb559280-bfa4-11e9-8964-686183dbae65.PNG)

in line 3 you look sdk 33.0.1.220

but when I run adl.exe

![adl_cp](https://user-images.githubusercontent.com/14032996/63125282-31f29000-bfa5-11e9-8b51-9681a259e202.PNG)

you see version 32.0.0.116

what's the problem

**Comment by ajwfrost** [comment]
Yes, so we have updated the SDK and the runtime version for Android so far - this is at version 33.

The desktop builds are not yet updated so these are still at version 32. ADL uses the desktop runtime file so will show as 32 still.

This is also why the namespace needs to be kept at 32.0 at the moment: we could update this (and the swf version) for Android only, but then the toolchain would break as most people (everyone?!) would be using ADL with a desktop runtime, and this throws an error if you try to use a namespace that it doesn't know about.

For now then, please continue to use 32.0 as your application descriptor namespace, this is not an error and will not cause any real problems..

thanks

**Comment by jeuxdaily** [comment]
hello
thanks for you ANE

I'm using ANE milkmanplugins .... https://www.milkmanplugins.com ... and I' have updated to supporte 64bit

without result

**Comment by ajwfrost** [comment]
I don't think that Milkman ANEs are supported any more, I've not seen any hint that they're updating these to support 64-bit Android versions. So you may need to switch to another ANE provider I'm afraid..

**Comment by jeuxdaily** [comment]
thanks

I see that

it's only for android that not support 64bit

but In IOS woek fine 

how can i do now

I need to change to another plugins

thanks
